libraries/rush-lib/src/api/LastInstallFlag.ts

```diff
@@ -12,6 +12,14 @@ const lodash: typeof import('lodash') = Import.lazy('lodash', require);
 
 export const LAST_INSTALL_FLAG_FILE_NAME: string = 'last-install.flag';
 
+/**
+ * @internal
+ */
+export interface ICheckNpmrcOptions {
```

Comment:
Hmm, given that this is being passed into the lockfile `isValid` methods, it feels a bit out of place to reference NPMRC.... maybe there should be a new `ILockfileValidityOptions` or something like that, which contains an `npmrcOptions` field? Also, the `rushVerb` property feels out of place in this interface but wouldn't in `ILockfileValidityOptions`

Reply:
Oops. I had npmrc on the mind when I wrote that interface name. Lemme change that.